rts/idris_rts.c

```diff
@@ -7,6 +7,11 @@
 #include "idris_bitstring.h"
 #include "getline.h"
 
+#define STATIC_ASSERT(COND,MSG) typedef char static_assertion_##MSG[(COND)?1:-1]
+
+
+STATIC_ASSERT(sizeof(Closure) == 16, sizeofClosure);
```

Comment:
This could be false on 32-bit systems depending on how double is aligned.

Reply:
I guess a 32-bit "test_c" job in the CI would be nice... I have a 32-bit Nix system at work, I can try there anyway.